modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -19,7 +19,8 @@ def initialize(info = {})
       'Author'         =>
         [
           'JaGoTu',
-          'zerosum0x0'
+          'zerosum0x0',
+          'Tom Sellers'
```

Comment:
Should also credit The UK's National Cyber Security Centre (NCSC) for finding the bug.

Reply:
Took care of this.